package.json

```diff
@@ -50,7 +50,7 @@
     "@typescript-eslint/eslint-plugin": "^6",
     "@typescript-eslint/parser": "^6",
     "aws-cdk": "^2",
-    "aws-cdk-lib": "2.28.0",
+    "aws-cdk-lib": "2.85.0",
```

Comment:
Any particular reason we's specifying a version that's so old?

Reply:
The API for AwsCustomResource doesn't let you pick the node.js runtime version.

It was upgraded from 12.x to 14.x with PR #20595 in version 2.28

It was upgraded from 14.x to 16.x with PR #24916 in version 2.77. There were some leftovers that were fixed in version 2.85 with PR #25995

Since 16.x runtime is still allowed till March 31, 2025 as per https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html it is the min version to be used

Reply:
Thanks for providing the context. Discussed with team member that 2.85 is the minimal version that supports node16 as the runtime for custom resource framework so 2.85 should be good to go too.